src/main/java/com/dynatrace/jenkins/dashboard/utils/Utils.java

```diff
@@ -61,15 +61,18 @@ private Utils() {
 
 	public static DynatraceClient createClient() {
 		final TAGlobalConfiguration globalConfig = GlobalConfiguration.all().get(TAGlobalConfiguration.class);
-		BasicServerConfiguration config = new BasicServerConfiguration(globalConfig.username,
-				globalConfig.password,
-				globalConfig.protocol.startsWith("https"),
-				globalConfig.host,
-				globalConfig.port,
-				globalConfig.validateCerts,
-				//connection timeout, 0 stands for infinite
-				0);
-		return new DynatraceClient(config);
+		if (globalConfig != null) {
```

Comment:
check your formatter - the code looks ugly now, but maybe GH makes some formatting while rendering the page

Reply:
I think it's Github formatting.